### ISSUE TITLE
Windows version does not zero-terminate strings

### ISSUE THREAD

**Issue by Cirromulus** [mortbopet/Ripes#84]
Hi!
I have the issue that, under windows 10, strings with `length % 4 != 0` are not null-terminated.
This leads to printing two adjacent strings.
This does not happen on my linux machine.

The following code produces two different binaries:

```riscv-asm
.data
n: .word 25		# Hier Wert für n eintragen
root: .word 5		# Hier das erwartete Ergebnis eintragen

strCalc: .string "Berechne Quadratwurzel von "
strErg: .string ". Das Ergebnis "
strIs: .string " ist "
strKorr: .string "korrekt."	# If len % 4 == 0, no zero byte is added
strFals: .string "falsch."
```
Windows version:

![image](https://user-images.githubusercontent.com/7480344/116078796-54541080-a697-11eb-9999-921ed949d5b9.png)

Linux (or on windows with extra space-character):
![image](https://user-images.githubusercontent.com/7480344/116078901-78175680-a697-11eb-97a0-3cdb77d821e2.png)

I am using v2.1.0 on both machines.

**Comment by mortbopet** [comment]
hi @Cirromulus, thank you for raising the issue. It should be fixed in 2aa04f7 and the fix is available in the [most recent continuous release](https://github.com/mortbopet/Ripes/releases/tag/continuous), if you'd like to test/verify.

**Comment by Cirromulus** [comment]
It works with the continuus release version.
Thanks!
